tests/RobotFramework/tests/tedge_agent/workflows/inner_command.toml

```
@@ -0,0 +1,15 @@
+operation = "sub_command"
```

Minor:
```suggestion
operation = "inner_command"
```
Or rename the file to `sub_command.toml ` just for consistency while referring to this workflow.